### ISSUE TITLE
Frequent crashes on vivoactive 3

### ISSUE THREAD

**Issue by axcro7** [warmsound/crystal-face#110]
I like this watchface a lot, but it frequently crashes and the watch returns to the default face. I then can't swap back, the preview only shows an [iQ!] Symbol and can't be selected.
No idea if it's an issue with the app or with the watch itself..

**Comment by warmsound** [comment]
Thanks for reporting this! What firmware are you running? At last update, did you uninstall the previous version before installing the new version of Crystal? If you can supply some logs (`/GARMIN/APPS/LOGS/CIQ_LOG.yml` on the watch), I can confirm whether you are seeing the #83 issue, which I'm hoping Garmin will release a firmware update for.

**Comment by axro7** [comment]
Hi again! 
as said in #109 I reinstalled the watchface today. So far no crashes.
The FW version is 6.10
In the logs there only was this old entry:
```
---
Error: Array Out Of Bounds Error
Details: 'Failed invoking <symbol>'
Time: 2018-11-11T17:55:44Z
Part-Number: 006-B2700-00
Firmware-Version: '5.50'
Language-Code: deu
ConnectIQ-Version: 3.0.4
Store-Id: 9fd04d09-8c80-4c81-9257-17cfa0f0081b
Store-Version: 33
Filename: 8ARD4729
Appname: Crystal
Stack:
  - pc: 0x10004fdf
  - pc: 0x10004a2c
  - pc: 0x10000041
```

**Comment by warmsound** [comment]
Thanks for getting back to me on this. OK, so it looks like the issues you originally reported are different from #83, and that so far, it appears a reinstallation has fixed it. Strangely, there were no logs for these recent crashes, so it's difficult to work out what was going on there. I wish I could understand this, so I could determine if there's anything I can do from my end to minimise the need to reinstall. I think it's safe to close this issue now.

**Comment by axcro7** [comment]
Hi again,
I can confirm that with the weather field activated the crashes are far more frequent.
I turned the field off, but the crashes do still occur. 
Unfortunately there are no logfiles. Is there a way to turn logging on or something?
All I can supply is how it looks like when the watchface crashes and I want to re enable it:
![image](https://user-images.githubusercontent.com/33942658/57008606-9570d180-6bf1-11e9-8ca0-a3ccb5ff064b.png)

Concerning the reinstallation: I noticed that the version numbering never changes in the settings section on my Android phone until reinstall Crystal.

Greetings, Alex
